src/content/staking/withdrawals/index.md

```diff
@@ -0,0 +1,186 @@
+---
+title: Staking withdrawals
+description: Page summarizing what staking push withdrawals are, how they work, and what stakers need to do to get their rewards
+lang: en
+template: staking
+image: ../../../assets/staking/leslie-withdrawal.png
+alt: Leslie the rhino with her staking rewards
+sidebarDepth: 2
+summaryPoints:
+  - The Shanghai upgrade brings staking withdrawals to Ethereum
+  - Validator operators must provide a withdrawal address to enable
+  - Rewards payments are automatically sent to the address provided every few days
+  - Fully exited validators will automatically receive their remaining balance
+---
+
+<UpgradeStatus dateKey="page-upgrades-withdrawals">
+  Staking withdrawals will be enabled through the Shanghai/Capella upgrade. This upgrade is expected to take place in the first half of 2023. <a href="#when">More below</a>
+</UpgradeStatus>
+
+The Shanghai/Capella upgrade enables **staking withdrawals** on Ethereum, allowing people to unlock rewards and optionally fully withdrawal funds from staking. Rewards payments will automatically and regularly be sent to a provided withdrawal address linked to each validator. Users can also exit staking entirely, unlocking their full validator balance.
```

Comment:
The Shanghai/Capella Ethereum upgrade will enable **staking withdrawals**. Staking withdrawals will allow validators to fully withdraw their deposited funds or withdraw the excess balance above 32 ETH.

Reply:
Hey @0xMNG! Thanks for hoping in to help out.. I'll take a look through and see what we can incorporate.

Think we can use part of this in the next iteration, but we were looking to trim out the comment about full withdrawals towards the beginning since this is mentioned explicitly at the end.

Reply:
Left a couple comments, pulled one suggestion in... a handful of them I felt were more semantic adjustments and wasn't sure they were adding too much, but I'm biased =) so I'll leave them for others to comment. Thank you for helping out!

More adjustments will continue to roll out here